integreat_cms/cms/models/poi_categories/poi_category.py

```diff
@@ -0,0 +1,53 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from ..abstract_base_model import AbstractBaseModel
+
+
+class POICategory(AbstractBaseModel):
+    """
+    Data model representing a POI category.
+    """
+
+    category_name = models.CharField(
+        max_length=250,
+        blank=False,
+        verbose_name=_("category name"),
+        help_text=_("The name of the POI category."),
+    )
+    english_category_name = models.CharField(
+        max_length=250,
+        blank=False,
+        verbose_name=_("english category name"),
+        help_text=_("The english name of the POI category."),
+    )
```

Comment:
```suggestion
```



I'm not sure how that's supposed to work. Since the categories should be used in the apps, it's not sufficient to translate them into German and English. Instead, we need one translation per language that exists in the CMS. In other words, we probably need an additional translation model which holds two foreign keys - one to the `POICategory`, one to the `Language` model, and one field which contains the translated category name.
In the CMS, we should probably show all those translated fields one one page, so one doesn't have to reload the page for every single translation in other languages ([formsets](https://docs.djangoproject.com/en/3.2/topics/forms/formsets/) is the keyword here).

```
class POICategoryTranslation(AbstractBaseModel):
    """
    Data model representing a POI category translation.
    """

    category = models.ForeignKey(
        POICategory,
        on_delete=models.CASCADE,
        verbose_name=_("category"),
    )
    language = models.ForeignKey(
        Language,
        on_delete=models.CASCADE,
        verbose_name=_("language"),
    )
    name = models.CharField(
        max_length=250,
        blank=False,
        verbose_name=_("category name"),
        help_text=_("The name of the POI category."),
    )
```